README.md

```diff
@@ -1,2 +1,19 @@
-# hw01
-Repository for homework 01
+# Homework One
+*by Gabriel Kahn*
+
+Hello my name is Gabriel Kahn and I am **very** excited to be taking this course with all of you. There are lots of fun facts of abut me. Let's make a list:
+
+* I am a MAPSS Student
+    + My concentration is Political Science
+* I enjoy Adam Sandler movies
+* My favorite food is spaggheti
+
+Additionally, here is a picture of my favorite animal
+
+![Tiger](http://goodnature.nathab.com/wp-content/uploads/2014/07/7465214366_5d0cb50707_k.jpg)
+
+That's enough facts for now, but there are many more.
+
+Here is an excellent [site](https://cfss.uchicago.edu) if you want to learn more about how I created this document.
+
+
```

Comment:
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   x        |              |            |
| **Coding strategy**         |       x    |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |      x     |              |            |

I think it's great that you used all 5 elements of markdown, I didn't get the links to work as I should have apparently!  Your tiger picture is also really awesome.  Really great job, I can't find anything to criticize.